src/app/globals.css

```
@@ -80,37 +79,43 @@
+    filter: drop-shadow(0 0 10px rgba(255, 106, 0, 0.8))
+      drop-shadow(0 0 20px rgba(255, 106, 0, 0.6))
+      drop-shadow(0 0 30px rgba(255, 90, 0, 0.4));
   }
 
   .text-glow-stroke-neon {
-    font-family: "Varela Round", sans-serif;
+    font-family: var(--font-varela-round), sans-serif;
     --color: #ffffffcf;
     --stroke: #ffffff;
     --blur: 5rem 133px;
     letter-spacing: 2px;
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)